src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
```

Comment:
Variable naming like 'result' decreases code readability. Consider using a more descriptive name such as 'styleObject' to represent the CSS declarations as an object.